WISE-Community/WISE
# Project folder name now matches project id

Newly created project's id and folder name should match.

Test the following for W4 and W5:
1. create project
2. copy project
3. create run
4. import project

make sure that the project id matches the name of the folder for the project contents.

Closes #1963
